app/services/patient_import/spreadsheet_transformer.rb

```diff
@@ -266,6 +266,23 @@ def medication_by_name_dosage_and_frequency(name)
       end
     end
 
+    def gender(value)
+      case value.downcase
+      when "m", "male"
+        "male"
+      when "f", "female"
+        "female"
+      when "t", "transgender"
+        "transgender"
+      else
+        value
```

Comment:
Given that this is a data import, do we want to have an else case that allows the value to pass through? Will that result in unexpected values in the database?

Reply:
That will let the bad data hit our model validations, which is already hooked up to provide feedback back to the user in a flash message.